dwcj-engine/src/main/java/org/dwcj/component/textarea/TextArea.java

```diff
@@ -1077,7 +1183,18 @@ protected void catchUp() throws IllegalAccessException {
 
     if (Boolean.FALSE.equals(this.tabTraversable)) {
       this.setTabTraversable(this.tabTraversable);
+    }
+
+    if (!this.label.equals("")) {
+      this.setLabel(this.label);
+    }
+
+    if (!this.placeholder.equals("")) {
+      this.setPlaceholder(this.placeholder);
+    }
 
+    if (Boolean.FALSE.equals(this.spellcheck)) {
```

Comment:
If the spellcheck option is false by default, we likely want to check if it's true, and then if so call the set method.